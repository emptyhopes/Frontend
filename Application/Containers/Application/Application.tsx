import styled from "styled-components";

import { Button } from "@/Application/Containers/UI/Button/Button";
// import { Routes } from "@/Application/Ship/Routes/Routes";

const StyledApplication = styled.div`
  *,
  *::after,
  *::before {
    box-sizing: border-box;
  }

  html,
  body {
    height: 100%;
  }

  body {
    padding: 0;
    margin: 0;
  }

  a {
    color: inherit;
    text-decoration: none;
  }

  a:hover {
    color: #ffffff;
    cursor: pointer;
  }

  li {
    list-style: none;
  }

  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 0;
  }
`;

const Application = () => {
  return (
    <StyledApplication data-testid="StyledApplication">
      <Button>12313</Button>
      {/* <Routes /> */}
    </StyledApplication>
  );
};

export { Application };
