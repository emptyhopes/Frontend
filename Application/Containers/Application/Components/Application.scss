// The file is intended to prevent stylelint from crashing with an error that no scss files were found

body {
  color: "#000000";
}
