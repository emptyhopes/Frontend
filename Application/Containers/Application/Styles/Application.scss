*,
*::after,
*::before {
  box-sizing: border-box;
}

html,
body {
  height: 100%;
}

body {
  padding: 0;
  margin: 0;
  background-color: black;
}

a {
  color: inherit;
  text-decoration: none;
}

a:hover {
  color: #ffffff;
  cursor: pointer;
}

li {
  list-style: none;
}

h1,
h2,
h3,
h4,
h5,
h6 {
  margin: 0;
}
