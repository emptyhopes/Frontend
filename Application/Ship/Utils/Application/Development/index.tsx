import * as React from "react";
import * as ReactDOM from "react-dom/client";

import { Application } from "@/Application/Containers/Application/Application";

const Structures = <Application />;

const element = document.getElementById("root");
const root = ReactDOM.createRoot(element);

root.render(Structures);
