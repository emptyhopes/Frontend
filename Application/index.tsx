import * as React from "react";
import * as ReactDOM from "react-dom";

// import { BrowserRouter } from "react-router-dom";

import { Application } from "@/Application/Containers/Application/Application";

// if ("serviceWorker" in navigator) {
//   window.addEventListener("load", () => navigator.serviceWorker.register("ServiceWorkers.js"));
// }

// const Structures = (
//   <React.StrictMode>
//     <BrowserRouter>
//       <Application />
//     </BrowserRouter>
//   </React.StrictMode>
// );

const Structures = <Application />;

const root = document.getElementById("root");

ReactDOM.hydrate(Structures, root);
