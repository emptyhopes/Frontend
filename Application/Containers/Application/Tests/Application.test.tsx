import { render, screen } from "@testing-library/react";

import { Application } from "@/Application/Containers/Application/Application";

describe("Application", () => {
  test("Render", () => {
    render(<Application />);

    const StyledApplication = screen.getByTestId("StyledApplication");
    expect(StyledApplication).toBeInTheDocument;
    expect(StyledApplication).toMatchSnapshot();
  });
});
