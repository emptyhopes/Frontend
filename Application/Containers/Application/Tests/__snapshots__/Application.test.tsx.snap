// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application Render 1`] = `
<div
  class="sc-gswNZR jTerpc"
  data-testid="StyledApplication"
>
  <button
    class="sc-bcXHqe WUNSp"
  >
    12313
  </button>
</div>
`;
